packages/2019-education/src/common/DataCollaborativeVictoryTheme.js

```diff
@@ -0,0 +1,313 @@
+import { assign } from "lodash";
```

Comment:
This should be imported from the component library rather than recreated unless it's style is different (which I can't tell off-hand) or we want for this package to avoid brand / visualization updates that the rest of the site receives.